app/brave_settings_strings.grdp

```diff
@@ -1391,6 +1391,22 @@
     Disable mute function in tab speaker icons
   </message>
 
+  <!-- Settings / Leo Assistant -->
+  <message name="IDS_SETTINGS_LEO_ASSISTANT" desc="Title for an item in the 'Leo Assistant' section of Brave Settings">
+    Leo Assistant
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_SHOW_ICON_ON_TOOLBAR_LABEL" desc="The text for settings option">
+    Show Leo assistant icon in the toolbar
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_SHOW_SUGGESTED_PROMPTS_LABEL" desc="The text for settings option">
+    Show suggested prompts in the conversation
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_RESET_AND_CLEAR_DATA_LABEL" desc="The text for settings option">
+    Reset and clear Leo Assistant data
+  </message>
```

Comment:
This is a bit confusing, what's getting reset? Maybe just "Clear Leo data"? cc: @rmcfadden3

Reply:
changed here: https://github.com/brave/brave-core/pull/19212/commits/0498497d0dc0b299b899bea60bfe994ef5265f1a